src/js/backend.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+(function() {
+  var URL = 'http://localhost:7777/kekstagram/data';
+
+  window.upload = function(data, onSuccess) {
+    var xhr = new XMLHttpRequest();
+    xhr.responseType = 'json';
+
+    xhr.addEventListener('load', function() {
+      onSuccess(xhr.response);
+    });
+
+    xhr.open('POST', URL);
+    xhr.send(data);
+  };
+
+  window.load = function(onSuccess) {
```

Comment:
```suggestion
  window.load = function(onSuccess, onError) {
```
думаю надо бы применить те же замечания, что и в учебном проекте в личном, не буду их все копировать сюда